tool/template-home.html

```diff
@@ -112,7 +112,7 @@ <h6 class="section-marker">{% trans %}Sustainability{% endtrans %}</h6>
       </div>
       <div class="col-sm-6 offset-sm-1">
         <h3 class="h1 mb-9">{% trans %}Green by Nature{% endtrans %}</h3>
-        <p class="mr-sm-5">{% trans %}XRP was designed with sustainability in mind. Contrary to other blockchains that use proof-of-work, the XRP Ledger confirms transactions through a unique “<a href="intro-to-consensus.html">consensus</a>” mechanism that consumes negligible energy.{% endtrans %}</p>
+        <p class="mr-sm-5">{% trans %}XRP was designed with sustainability in mind. Contrary to other blockchains that use proof-of-work, the XRP Ledger confirms transactions through a unique “<a href="intro-to-consensus.html">consensus</a>” mechanism and is one of the first truly carbon neutral blockchains.{% endtrans %}</p>
```

Comment:
"one of"? What other blockchains are even close?

Reply:
Also, I feel like it would be appropriate to link to a news or blog article that explains _how_ the XRPL is carbon-neutral (i.e. [XRP Ledger Foundation is buying](https://ripple.com/ripple-press/ripple-leads-sustainability-agenda-to-achieve-carbon-neutrality-by-2030/) [EW Zero credits](https://www.energyweb.org/technology/applications/ew-zero/) to offset energy usage.

Reply:
@mDuo13 - brought the points up to Polina. 

- Changing the copy on the Impact Page 
- Adding links to both the Impact and Home Page, per your suggestion